Orange/widgets/regression/owlinearregression.py

```diff
@@ -27,12 +27,13 @@ class OWLinearRegression(OWProvidesLearner, widget.OWWidget):
                ("Coefficients", Table)]
 
     #: Types
-    OLS, Ridge, Lasso = 0, 1, 2
+    OLS, Ridge, Lasso, Elastic = 0, 1, 2, 3
 
     learner_name = settings.Setting("Linear Regression")
     ridge = settings.Setting(False)
     reg_type = settings.Setting(OLS)
     alpha_index = settings.Setting(0)
+    l1_ratio = settings.Setting(0.5)
```

Comment:
This looks like a sensible default.
Could you also change the above line to something other than the smallest possible value (e.g. the index of 0.5 or 1).